openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py

```diff
@@ -19,10 +18,11 @@ def update_course_settings(*args, **kwargs):  # pylint: disable=unused-argument
         other_course_settings = kwargs.get('other_course_settings', {})
 
         OtherCourseSettings.objects.update_or_create(
-            course_overview=CourseOverview.get_from_id(kwargs.get('course_key')),
+            course_id=kwargs.get('course_key'),
             defaults={
                 'external_course_id': other_course_settings.get('external_course_run_id', ''),
                 'external_platform': other_course_settings.get('external_platform_target', ''),
```

Comment:
Same as the line above: 
```suggestion
                'external_platform': other_course_settings.get('external_platform_target'),
```

Reply:
Applied.